src/core/components/auth/auths.jsx

```diff
@@ -109,15 +109,14 @@ export default class Auths extends React.Component {
             <p>API requires the following scopes. Select which ones you want to grant to Swagger UI.</p>
           </div>
             {
-              definitions.filter( schema => schema.get("type") === "oauth2")
-                .map( (schema, name) =>{
+              oauthDefinitions.map( (schema, name) =>{
```

Comment:
`oauthDefinitions` is defined above as `definitions.filter( schema => schema.get("type") === "oauth2")` so repeating that code here was redundant. I stumbled upon this during my research so I fixed it... Should maybe have been a separate PR.